### ISSUE TITLE
Option "increaseHarvestingTimePerLog" doesn't work

### ISSUE THREAD

**Issue by HySand** [ricksouth/serilum-mc-mods#1534]
## **Information**
Minecraft version: 1.19.2
Modloader: Fabric
Fabric loader version: 0.14.12
Environment: Singleplayer & Multiplayer

Mod name: Tree Harvester
Mod version: 5.9

## **Description**

```	
"increaseHarvestingTimePerLog": true,
"increasedHarvestingTimePerLogModifier": 1.0,
```
It seems nothing changes by setting these.

Here is the full json file

<details>
<summary> treeharvester-fabric.json5 </summary>

```
{
	// If enabled, tree harvesting only works when a player is holding an axe in the main hand.
	"mustHoldAxeForTreeHarvest": true,
	// If enabled, tree harvesting works when not holding the sneak button. If disabled it's reversed, and only works when sneaking.
	"treeHarvestWithoutSneak": true,
	// If enabled, players instantly break the leaves as well as all logs of the tree when a bottom log is broken.
	"instantBreakLeavesAround": false,
	// Whether the mod should attempt to find the actual bottom log of the tree and start there. This means you can break a tree in the middle and it will still completely be felled.
	"automaticallyFindBottomBlock": false,
	// If enabled, the leaves around a broken tree will quickly disappear. Only works with 'instantBreakLeavesAround' disabled.
	"enableFastLeafDecay": true,
	// If enabled, the warped stem/crimson trees in the nether will also be chopped down quickly.
	"enableNetherTrees": true,
	// If enabled, giant/huge mushrooms will also be chopped down quickly.
	"enableHugeMushrooms": true,
	// If enabled, automatically replaces the sapling from the drops when a tree is harvested.
	"replaceSaplingOnTreeHarvest": false,
	// If enabled, automatically replaces the sapling from the drops when a huge mushroom is harvested and 'enableHugeMushrooms' is enabled.
	"replaceMushroomOnMushroomHarvest": false,
	// If enabled, for every log harvested, the axe held loses durability.
	"loseDurabilityPerHarvestedLog": true,
	// Here you can set how much durability chopping down a tree should take from the axe. For example if set to 0.1, this means that every 10 logs take 1 durability.
	// min: 0.001, max: 1.0
	"loseDurabilityModifier": 1.0,
	// If enabled, players' exhaustion level increases 0.005 per harvested log (Minecraft's default per broken block) * increaseExhaustionModifier.
	"increaseExhaustionPerHarvestedLog": true,
	// This determines how much exhaustion should be added to the player per harvested log. By default 0.005 * 1.0.
	// min: 0.001, max: 1.0
	"increaseExhaustionModifier": 1.0,
	// If enabled, harvesting time will increase per existing log in the tree. The amount is determined by 'increasedHarvestingTimePerLogModifier'.
	"increaseHarvestingTimePerLog": true,
	// How much longer it takes to harvest a tree with 'increaseHarvestingTimePerLog' enabled. The actual speed is: newSpeed = originalSpeed / (1 + (logCount * increasedHarvestingTimePerLogModifier)).
	// min: 0.01, max: 10.0
	"increasedHarvestingTimePerLogModifier": 1.0,
	// How many leaves should be broken per tick after a tree has been harvested. Increasing this will speed up the fast leaf decay, but costs more processing power per tick.
	// min: 1, max: 16
	"amountOfLeavesBrokenPerTick": 1
}
```
</details>

**Comment by christoros** [comment]
I am experiencing the same problem.

**Comment by ricksouth** [comment]
Thanks for opening the issue and giving the mod a try! This has been fixed in version 6.48 of Collective. Tree Harvester doesn't need an update.

https://www.curseforge.com/minecraft/mc-mods/collective/files
